charts/camunda-platform-8.6/values-latest.yaml

```diff
@@ -23,7 +23,7 @@ connectors:
 operate:
   # https://hub.docker.com/r/camunda/operate/tags
   image:
```

Comment:
We still need to set the `repository` explicitly here.

Reply:
Thanks for the review - and sorry - I dont get what I need to set 🙈

Reply:
Ah I think I got it now looking at the other values 💡 Let me prepare a commit

Reply:
Under Console and Operate section, we need to add the repo name like we do for the rest of the components.

Here is how we do it in Optimize for example:
https://github.com/camunda/camunda-platform-helm/blob/fedd289871573acd49315daf023ab5286e9c6db9/charts/camunda-platform-8.6/values-latest.yaml#L31


Also, we need to ensure that it's applied to all `values-latest.yaml` (from 8.3 to 8.6 and alpha).
The same issue could be there, too.

Reply:
@aabouzaid done in the last commit.

BTW, I also realized that `repository` was missing for the webModeler and the version was actually one higher than stated in the version matrix. Should I push that change too? 

<img width="1828" alt="image" src="https://github.com/user-attachments/assets/71a924ed-9f28-4587-9d90-ea8e8cf29a2a" />

Reply:
There are even more missing the repository flag. Should I set it to all (zeebe, tasklist, etc... )?

Reply:
the goal is for renovate to update the dependencies. The web-modeler images should be kept up-to-date given that it has the prefix 
```
# renovate: datasource=docker depName=camunda/web-modeler-restapi
```
so that section does not need the `repository` specified. I've updated the others.

Reply:
I've set all of these.

Reply:
Thank you!